en/identity-server/next/docs/deploy/configure/databases/carbon-database/change-to-mysql.md

```diff
@@ -78,6 +78,29 @@ A sample configuration is given below.
         password = "regadmin"
         ```  
 
+
+    !!! note
+        When integrating WSO2 Identity Server with MySQL DB, it is advisable to deactivate the case-insensitive functionality within WSO2 IS.
+        This recommendation is based on the inherent case-insensitivity of MySQL DB, which renders the similar feature in WSO2 IS redundant.
+        Disabling this feature in WSO2 IS eliminates unnecessary processing, thereby potentially enhancing overall system performance.
+
+        To disable case-insensitivity, use the following configurations.
+
+        For Primary Userstores
```

Comment:
Add the file  locations similar to the MSSQL note. (Replicate the same)

Reply:
Fixed with https://github.com/wso2/docs-is/pull/4209/commits/d0b97b1a597730a4d099c4dc1565e31452c0f03f